example/home/models.py

```diff
@@ -162,6 +163,11 @@ class BlogPageRelatedLink(Orderable):
 
 
 @register_snippet
+@register_singular_query_field("person", {"name": graphene.String()})
+@register_plural_query_field("people", {"job": graphene.String()})
+@register_plural_query_field(
```

Comment:
@zerolab on the other hand it may be nice to separate @register_singular_query_field and @register_plural_query_field to make it simpler to document. Decorators in my experience tend to do discrete things. Having one decorator that sets up two different fields means we need to explain how both behave under on heading.  Where as if we had both individual decorators, then the aggregate decorator can be explained by linking to both... just a thought...  Maybe not something to include in this fix, but something to think about long term.